src/GuestConfiguration/GuestConfiguration/Az.GuestConfiguration.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.10.2'
+ModuleVersion = '0.10.3'
```

Comment:
@aashish1987 you can revert this change since we'll do the versioning before the module is released